### DISCUSSION TITLE
face swap failed

### DISCUSSION THREAD

**Discussion by coffeedrinker0725** [lllyasviel/stable-diffusion-webui-forge#418] [Q&A]
The latest updated version corrected one of the tuple error, but another tuple error remained: FaceSwapLab - ERROR - Failed to swap face in postprocess method : 'tuple' object has no attribute 'height'. This happened when testing face swap in img2imge mode.

**Comment by catboxanon** [comment]
The issue you listed is identical to the one mentioned in https://github.com/lllyasviel/stable-diffusion-webui-forge/discussions/409.
